### ISSUE TITLE
Update for Docusaurus 2.2.0

### ISSUE THREAD

**Issue by FHannes** [typesense/docusaurus-theme-search-typesense#24]
Docusaurus 2.2.0 has been released. Version 0.7.0-0 of the plugin is not compatible due to dependency conflicts.

Maybe it is possible to specify a minimum version for the docusaurus dependencies, so there does not have to be an update of the plugin for every release? I don't know how stable the docusaurus apis are...

**Comment by jasonbosco** [comment]
I just upgraded the sandbox project I have to use 0.7.0-0 of the typesense theme and it works fine for me: https://github.com/typesense/docusaurus-typesense-testground/commit/10141b6e575f3fcacd4b3dcb230dfc5feb5ea974

Could you try running that locally?

**Comment by dustinboss** [comment]
This was an issue for me, too. NPM failed to install `docusaurus-theme-search-typesense@0.7.0-0` into a fresh install of `@docsaurus/core@2.2.0`. 

The error was: 
```
Could not resolve dependency:
peer @docusaurus/core@"2.1.0" from docusaurus-theme-search-typesense@0.7.0
```

I got around it with `npm install docusaurus-theme-search-typesense@next --force` but it still threw warnings about the dependency issues.

**Comment by dustinboss** [comment]
Update: the error doesn't seem to happen in Yarn, but it does with NPM

**Comment by FHannes** [comment]
Yes, with NPM there seem to be issues, also with the test project:
```
docusaurus-typesense-testground$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: docusaurus-typesense-testground@0.0.0
npm ERR! Found: @docusaurus/core@2.2.0
npm ERR! node_modules/@docusaurus/core
npm ERR!   @docusaurus/core@"2.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @docusaurus/core@"2.1.0" from docusaurus-theme-search-typesense@0.7.0-0
npm ERR! node_modules/docusaurus-theme-search-typesense
npm ERR!   docusaurus-theme-search-typesense@"^0.7.0-0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/fhannes/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/fhannes/.npm/_logs/2022-11-15T09_53_52_522Z-debug-0.log
```

I am also getting errors at runtime if I use the force to install with NPM: [typesense-errors.log](https://github.com/typesense/docusaurus-theme-search-typesense/files/10011514/typesense-errors.log)

**Comment by jasonbosco** [comment]
Awesome, thank you for confirming. I've published `0.8.0`.

**Comment by FHannes** [comment]
Thank you for the swift resolution! Release 0.8.0 seems to address all of my issues. 🎉
